addon/components/bootstrap-checkbox-group.js

```diff
@@ -13,4 +13,18 @@ export default Ember.Component.extend(InputableMixin, {
   label: null,
 
   required: false,
+
+  actions: {
+    onChange(option, isChecked) {
+      const selected = this.get('selected');
```

Comment:
Also, you probably actually want `Ember.A(this.get('selected'))` as `makeArray` is private.